styles/variables-electro.less

```diff
@@ -149,6 +149,13 @@
 @agate-popup-closebutton-position-left: auto;
 @agate-popup-closebutton-position-right: 18px;
 
+// ProgressBar
+// ---------------------------------------
+@agate-progressbar-height: 10px;
```

Comment:
How do you feel about making Electro and Titanium taller (and different heights)? Titanium could be the same height as the Slider while Electro could be somewhere in the middle.

Reply:
Also, if small progress bars are a design requirement for the dashboard, maybe it's time to add a `small` prop to progress bar, which applies the `.small` class to get a shorter height option?